mq/README.md

```diff
@@ -0,0 +1,110 @@
+# Lazy Amazon MQ Module
+
+Module to deploy Amazon MQ broker where supported engine type is RabbitMQ
+Assumptions:
+- Amazon MQ will be created as a public broker until communicaiton with USXIT is established
+- Once established Private VPC portion needs to be implemented
+
+## Resources
+
+- Amazon MQ Broker (RabbitMQ)
+
+## Input Variables
+
+- region
```

Comment:
We need to start making these into tables, 
ex https://github.com/variant-inc/lazy-terraform/tree/master/efs#input-variables